cmd/minishift/cmd/openshift/service.go

```diff
@@ -19,39 +19,27 @@ package openshift
 import (
 	"fmt"
 	"os"
-	"strings"
-	"text/template"
 
 	"github.com/docker/machine/libmachine"
 	"github.com/pkg/browser"
 	"github.com/spf13/cobra"
 
-	"github.com/minishift/minishift/pkg/minikube/cluster"
 	"github.com/minishift/minishift/pkg/minikube/constants"
 	"github.com/minishift/minishift/pkg/util/os/atexit"
+
+	"github.com/minishift/minishift/pkg/minishift/openshift"
 )
 
 var (
-	namespace          string
-	serviceURLMode     bool
-	serviceURLFormat   string
-	serviceURLTemplate *template.Template
-	https              bool
+	namespace      string
+	serviceURLMode bool
```

Comment:
Actually, this naming thing is already there for [`minishift console`](https://minishift.io/docs/interacting-with-openshift.html#console) 
If we had used `dashboard` instead of `console` for command then we could have used `console` across minishift to refer printing in shell/console. 

Also, if we are changing the name here anyway, then we need to do across the code for consistency.

WDYT @gbraad @LalatenduMohanty @hferentschik @praveenkumar ?

```
$ minishift console   # Open Openshift UI

$ minishift console --url   #  Print url in console
```